.drone.yml

```diff
@@ -558,8 +548,7 @@ steps:
     pull: always
     image: techknowlogick/xgo:go-1.15.x
     commands:
-      - apt update && apt -y install curl
```

Comment:
do not worry about this removed line. I added curl into base xgo image.

Reply:
Nice. Faster builds are always welcome.